packages/components/src/components/List/ListItem.tsx

```diff
@@ -31,32 +31,98 @@ const Item = styled.li<ItemProps>`
 
 const BulletWrapper = styled.div`
     flex: 0;
+    position: relative;
 `;
 
 const ContentWrapper = styled.div`
     flex: 1;
 `;
 
-const Circle = styled.div<{ $elevation: Elevation }>`
+const Circle = styled.div<{ $isOrdered: boolean }>`
     width: ${spacingsPx.md};
     height: ${spacingsPx.md};
     border-radius: 50%;
-    border: ${spacingsPx.xxs} solid ${mapElevationToBorder};
+    border: ${spacingsPx.xxs} solid ${({ theme }) => theme.backgroundNeutralDisabled};
+
+    ${({ $isOrdered }) =>
+        $isOrdered &&
+        css`
+            position: relative;
+
+            &::before {
+                content: '';
+                position: absolute;
+                top: calc(100% + ${spacingsPx.xs});
+                left: 50%;
+                transform: translateX(-50%);
+                height: calc(var(--distance, 0) - ${spacingsPx.xs});
```

Comment:
pure magic :D

Reply:
Yeah, this is a little bit crazy but I really wanted it to work dynamically and not rely on the item to be one line:)